docs/source/modelbuilding.rst

```diff
@@ -54,4 +54,6 @@ This means that when an aircraft is being optimized for a mission, you can creat
 
 .. literalinclude:: examples/performance_modeling.py
 
+Note that the output table can be filtered with a list of variables to show.
```

Comment:
should drop this in the event that we accept my proposal to print `sol.table()` for this model